corehq/apps/enterprise/enterprise.py

```diff
@@ -383,3 +389,85 @@ def _get_individual_export_rows(self, exports, export_line_counts):
             )
 
         return rows
+
+
+class EnterpriseSMSReport(EnterpriseReport):
+    title = gettext_lazy('SMS Sent')
+    MAX_DATE_RANGE_DAYS = 90
+
+    def __init__(self, account, couch_user, start_date=None, end_date=None, num_days=30):
+        super().__init__(account, couch_user)
+
+        if not end_date:
```

Comment:
Can you make it as a util function so other tiles can reuse it?

Reply:
I believe I did this in the iterators PR, which is still in review. Re-adding it here would cause conflicts, so I'm going to wait until that PR is merged for this functionality